.github/workflows/reusable-native-tests.yml

```diff
@@ -32,5 +32,5 @@ jobs:
           # therefore we're starting a Kafka container manually for the tests
           docker compose -f .github/graal-native-docker-compose.yaml up -d
           # don't wait for startup - gradle compile takes long enough
-          ./gradlew nativeTest
+          ./gradlew -PtestLatestDeps=true nativeTest
```

Comment:
Since latest dep tests can fail without us doing anything wrong we have made it so that when latest dep test fails it won't prevent the PR form being merged. Perhaps we should make sure that native test failures are also treated that way.

Reply:
native tests are only run when you change something in spring - so maybe it's ok as is. 

alternatively, we could run the native tests twice - but it seems overkill

Reply:
> we could run the native tests twice

I think this makes sense for the daily native CI, it gives us coverage and confidence that we haven't broken the old, and that we work with the new

for PR build I'd say it would be ideal to only run against the old version for reason @laurit gives above